core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -517,6 +518,32 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with Timeou
     assertDataStructuresEmpty()
   }
 
+  test("unserializable partition") {
```

Comment:
The test "unserializable task" in 507 line only verify the task. This test should be retained, I will add a new test in CoarseGrainedSchedulerBackendSuite.